pkg/skaffold/schema/latest/config.go

```diff
@@ -1077,4 +1077,7 @@ type JibArtifact struct {
 	// `maven`: for Maven.
 	// `gradle`: for Gradle.
 	Type string `yaml:"type,omitempty"`
+
+	// BaseImage overrides the default jib base image.
```

Comment:
It's not just the default image (gcr.io/distroless/java:{8,11}).
```suggestion
	// BaseImage overrides the configured jib base image.
```